lib/livebook/notebook/learn/kino/intro_to_kino.livemd

```diff
@@ -55,8 +55,8 @@ the main way to interact with controls is by listening to their
 events.
 
 ```elixir
-Kino.Control.button("Click me!")
-|> Kino.listen(fn event -> IO.inspect(event) end)
+button = Kino.Control.button("Click me!") |> Kino.render()
+Kino.listen(button, fn event -> IO.inspect(event) end)
 ```
```

Comment:
````suggestion
button = Kino.Control.button("Click me!")
```

```elixir
Kino.listen(button, fn event -> IO.inspect(event) end)
```
````

Reply:
Thought this might be the suggestion, deal! 👍